adam/experiment/log_experiment.py

```diff
@@ -170,6 +176,19 @@ def learner_factory_from_params(
         return lambda: ObjectPursuitLearner.from_parameters(
             params.namespace("pursuit"), graph_logger=graph_logger
         )
+    elif learner_type == "pursuit-gaze":
+        return lambda: IntegratedTemplateLearner(
+            object_learner=PursuitObjectLearnerNew(
+                learning_factor=0.05,
+                graph_match_confirmation_threshold=0.7,
+                lexicon_entry_threshold=0.7,
+                rng=rng,
+                smoothing_parameter=0.002,
+                ontology=GAILA_PHASE_1_ONTOLOGY,
+                language_mode=language_mode,
+                rank_gaze_higher=True,
+            )
```

Comment:
Go ahead and add the rest of the learners here so if we want to run across the entire curriculum we can.